mixed_drinks/src/App.js

```diff
@@ -67,6 +68,7 @@ class App extends Component {
       <div className="App">
         <h1>Mixed Drinks</h1>
         <Input handleChange={this.handleChange} handleEnterPressed={this.handleEnterPressed} />
+        <AddDrinksButton />
```

Comment:
Was this component intended to be used with the Input component?  Currently the button says 
"Submit", so I am thinking this was created so a user can click submit and then the Card layouts update with the intended query.  Confirm if this is the functionality intended.